clouddriver-google/src/test/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/DestroyGoogleServerGroupAtomicOperationUnitSpec.groovy

```diff
@@ -234,6 +255,12 @@ class DestroyGoogleServerGroupAtomicOperationUnitSpec extends Specification {
         1 * computeMock.zoneOperations() >> zoneOperations
         1 * zoneOperations.get(PROJECT_NAME, location, INSTANCE_GROUP_OP_NAME) >> zoneOperationsGet
         1 * zoneOperationsGet.execute() >> instanceGroupManagersDeleteOp
+        registry.timer(
+            registry.createId("google.api",
+                  [api: "compute.instanceGroupManagers.delete",
```

Comment:
This map appears often enough in the tests that it might be worth it to create a helper method for the tests, something like: `Map generateMetricsTagsMap(String api, String scope)`